### DISCUSSION TITLE
How get machine learning service to work?

### DISCUSSION THREAD

**Discussion by OlegLoewen** [immich-app/immich#7122] [Q&A]
Please help me get the face detection and asset detection.

Issue:
If I navigate to 'Explore' area, I see only places, but no faces or assets.

I'm using Portainer to run immich in a stack with the following configuration
[immich_stack.txt](https://github.com/immich-app/immich/files/14169639/immich_stack.txt)
And the following ML environment variables: 
![image](https://github.com/immich-app/immich/assets/44769983/3c1e6cbf-89ed-48c2-9ee0-0865f6623ee5)

**Comment by bo0tzz** [comment]
Have the background jobs run? Can you post some more detail like logs etc?
